server/tests/common/data.py

```diff
@@ -343,8 +343,10 @@ async def wksp1_bob_becomes_owner_and_changes_alice(
         assert isinstance(outcome, RealmRoleCertificate)
 
 
-HttpCommonErrorsTesterDoCallback = Callable[[], Coroutine[None, None, None]]
-HttpCommonErrorsTester = Callable[[HttpCommonErrorsTesterDoCallback], Coroutine[None, None, None]]
+type HttpCommonErrorsTesterDoCallback = Callable[[], Coroutine[None, None, None]]
```

Comment:
:open_mouth:

Reply:
what does that type mean ?

Reply:
It defines a type alias: https://docs.python.org/3/library/typing.html#type-aliases

Reply:
My question was more about the callable and coroutine part

Reply:
Ah yes, it's corresponds to an async function with the following prototype
```python
async def my_func() -> None
```
It's equivalent to `Callable[[], Awaitable[None]]` which I find more readable.